src/libdredd/src/mutation_replace_binary_operator.cc

```diff
@@ -186,6 +205,48 @@ std::string MutationReplaceBinaryOperator::GetFunctionName(
   return result;
 }
 
+void MutationReplaceBinaryOperator::GenerateConstantInsertion(
+    std::stringstream& new_function, const clang::BuiltinType* exprType,
+    int& mutant_offset) const {
```

Comment:
It is good practice to put input-only parameters before output parameters:

https://google.github.io/styleguide/cppguide.html#Inputs_and_Outputs

Here, `new_function` is an output parameter as it gets mutated (as is `mutant_offset`), so it should come after the input parameters.

Also: you sould make `exprType` a const reference rather than a const pointer if that's workable - see the style link above for some good advice on this. Typically non-optional parameters should be references (so that you can't pass null).

Reply:
Done.